### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by Poltergeistz**
DL-4282

Removed the tags and keeping the git history as this is a copy from `frontend-public-decisions`.

- Renaming for worship-decisions.

- ~~Drone signature and secrets needs to be added for this project.~~ (Not needed since we use woodpecker now)

- Text copy might also be changed to match the app.

### CODE REVIEW

File: config/environment.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ module.exports = function (environment) {
         'acmidm-oauth2': {
           apiKey: '{{OAUTH_API_KEY}}',
           baseUrl: '{{OAUTH_BASE_URL}}',
-          scope: 'openid vo profile abb_besluitendatabank',
+          scope: 'openid vo profile abb_toezichtsdatabank',
```
Comment (Poltergeistz) [inline config/environment.js:35]: Do the scope needs to be changed for the right openid profile ?

File: package.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
-  "name": "frontend-public-decisions",
-  "version": "1.5.0",
+  "name": "frontend-worship-decisions",
+  "version": "0.1.0",
```
Comment (Windvis) [inline package.json]: I would set that to 0.0.0 for now. Sorry, I meant that the first tagged release will be `0.1.0` but we use release-it to bump that number automatically.

**Review by Windvis — COMMENTED** [review]
Some things that still need to be changed:

- https://github.com/lblod/frontend-worship-decisions/blob/d81ce8adea4f69361c3f7a8205988841635af3f9/app/components/contact-form.js#L26 It seems I missed this in the `app-name` refactor. I think we can also retrieve it from the config.environment similar to the app-name helper.
- https://github.com/lblod/frontend-worship-decisions/blob/d81ce8adea4f69361c3f7a8205988841635af3f9/app/templates/legaal/toegankelijkheidsverklaring.hbs#L6 This link needs to be updated, although I'm not sure what it is. If we don't know yet we should probably create an issue for it so we don't forget before the release.

File: package.json (hunk 2 above)
Comment (Poltergeistz) [reply package.json]: Ah yes no problem, I'm on it.

**Comment by Poltergeistz** [comment]
> Some things that still need to be changed:
> 
>     * https://github.com/lblod/frontend-worship-decisions/blob/d81ce8adea4f69361c3f7a8205988841635af3f9/app/components/contact-form.js#L26
>        It seems I missed this in the `app-name` refactor. I think we can also retrieve it from the config.environment similar to the app-name helper.
> 
Okay, do you know if the email will be the same ?
>     * https://github.com/lblod/frontend-worship-decisions/blob/d81ce8adea4f69361c3f7a8205988841635af3f9/app/templates/legaal/toegankelijkheidsverklaring.hbs#L6
>        This link needs to be updated, although I'm not sure what it is. If we don't know yet we should probably create an issue for it so we don't forget before the release.
Yes indeed, I haven't touched this part as I don't know what link we need to provide here.

**Comment by Windvis** [comment]
I'll merge this already since it's only some small details that we can also fix in a future PR.

File: config/environment.js (hunk 1 above)
Comment (cecemel) [reply config/environment.js:35]: You can change this abb_eredienstbesluitendatabank

Comment (cecemel) [reply config/environment.js:35]: will only be functional waaay later

**Comment by cecemel** [comment]
Except for the mini ACM context thing, we can go for a first release!

File: config/environment.js (hunk 1 above)
Comment (Poltergeistz) [reply config/environment.js:35]: Okay thanks :+1:

**Pull request merged**